docs/customresourcestate-metrics.md

```diff
@@ -305,6 +305,11 @@ spec:
           ...
 ```
 
+Produces:
+```prometheus
+uptime{group="myteam.io", kind="Foo", version="v1"} 43.21
+```
+
```

Comment:
@logicalhan JFYI I added a doc snippet here to make things a bit more explicit to the reader.

Reply:
sgtm